Doesn't work from within addons

## Version
ember-cli: 2.9.0-beta.2
node: 6.6.0
os: darwin x64
ember-cli-custom-assertions: 0.0.6

## Test Case
Not really a test case. 
https://github.com/DockYard/ember-cli-custom-assertions/issues/new

## Steps to reproduce
Install it in an empty addon and try to run `ember g assertion is-cool`.

## Expected Behavior
An assertion is generated

## Actual Behavior
Error: `Cannot find module 'ember-cli-string-utils'`

Given that I use this addon in apps without problem, seems the issue arises when used from within another addon.

--- a/package.json
+++ b/package.json
@@ -42,7 +42,8 @@
     "ember-addon"
   ],
   "dependencies": {
-    "ember-cli-babel": "^5.1.6"
+    "ember-cli-babel": "^5.1.6",
+    "ember-cli-string-utils": "^1.0.0"
   },
   "ember-addon": {
     "configPath": "tests/dummy/config"

